meshtastic/interfaces/ble/interface.py

```
@@ -672,130 +680,17 @@ def _invoke_safe_execute_compat(
-                exc_info=True,
-            )
-            _fallback_if_not_executed()
+        """Invoke safe_execute compatibility probing via notification dispatcher."""
+        BLENotificationDispatcher.invoke_safe_execute_compat(
+            safe_execute,
+            handler_thunk,
+            error_msg=error_msg,
+            fallback=fallback,
+        )
 
     def _from_num_handler(self, _: Any, b: bytes | bytearray) -> None:
```

Ruff (0.15.6)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_`

(ANN401)